deps/GMP/GMP.cmake

```diff
@@ -59,7 +59,8 @@ else ()
         URL https://github.com/SoftFever/OrcaSlicer_deps/releases/download/gmp-6.2.1/gmp-6.2.1.tar.bz2
         URL_HASH SHA256=eae9326beb4158c386e39a356818031bd28f3124cf915f8c5b1dc4c7a36b4d7c
         DOWNLOAD_DIR ${DEP_DOWNLOAD_DIR}/GMP
-        BUILD_IN_SOURCE ON 
+        PATCH_COMMAND git apply --verbose ${CMAKE_CURRENT_LIST_DIR}/0001-GMP_GCC15.patch
```

Comment:
It's better to add the `xxxx_DIRECTORY_FLAG` like https://github.com/SoftFever/OrcaSlicer/blob/dfdf9a31594a6365ab2ef0abff0613fd90e4c42a/deps/OpenCV/OpenCV.cmake#L14
so it can be build without a valid git repo.

Reply:
Fixed.

Reply:
I think you also need to define this flag, like https://github.com/SoftFever/OrcaSlicer/blob/dfdf9a31594a6365ab2ef0abff0613fd90e4c42a/deps/OpenCV/OpenCV.cmake#L7-L9

Reply:
Done